display.py

```diff
@@ -8,6 +7,7 @@
 
 class Terminal(luma_terminal):
     def __init__(self, device, font=None, color="white", bgcolor="blue", tabstop=4, line_height=None, animate=False, word_wrap=False):
+        # if > 0 then don't send image to device

```

Comment:
Don't understand the context here

Reply:
if `self.context_manager_depth > 0` then don't send image to hardware(don't flush, just update bitmap or however image is stored).
I could've written context manager for this class but it would be just doubling the one in ScreenDisplay.